recipe_scrapers/twopeasandtheirpod.py

```diff
@@ -1,12 +1,20 @@
 # mypy: disallow_untyped_defs=False
 from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
 
 
 class TwoPeasAndTheirPod(AbstractScraper):
     @classmethod
     def host(cls):
         return "twopeasandtheirpod.com"
 
+    def author(self):
+        author_tag = self.soup.find("p", class_="entry-author")
+        if author_tag:
+            return author_tag.text.replace("By ", "")
+
+        return "Two Peas & Their Pod"
```

Comment:
This is anecdotal based on checking a single recipe from the site, but it did look like author information is available as `schema.org` metadata on the page.  Does `self.schema.author()` not work here?

Reply:
The author info in the schema metadata seems to be
```
>>> scraper.schema.data["author"]
{'@id': 'https://www.twopeasandtheirpod.com/#/schema/person/6e69dc5468cedba1a1dbba42c05e2501'}
```
which I didn't think was very useful.

Reply:
Hrm, ok, yep - I'd only checked the example where a `name` field was also present.

It seems like the problem here is that we don't resolve entity references (such as the `@id` reference) in our `schema.org` data handling.

Retrieving the information from the rest of the HTML page source seems OK here - I'll file an issue to consider implementing additional `schema.org` entity cross-reference handling.

Reply:
I may be getting confused between `schema.org` and JSON-LD here; either way, in this case it appears that there's an indirect entity reference that would theoretically allow us to retrieve the data we want.